es/chain/node.js

```diff
@@ -105,7 +105,14 @@ async function tx (hash, info = true) {
 }
 
 async function height () {
-  return (await this.api.getCurrentKeyBlockHeight()).height
+  try {
+    if (this.height_query_promise === undefined) {
```

Comment:
this naming looks super unusual for js, shouldn't it be something like `_heightPromise`?